boot/seal_test.go

```diff
@@ -46,6 +46,14 @@ type sealSuite struct {
 
 var _ = Suite(&sealSuite{})
 
+func (s *sealSuite) SetUpTest(c *C) {
+	s.BaseTest.SetUpTest(c)
+
+	rootdir := c.MkDir()
+	dirs.SetRootDir(rootdir)
+	s.AddCleanup(func() { dirs.SetRootDir("/") })
```

Comment:
low-prio: now that we do this here, maybe some of the tests below don't need to setup a rootdir on their own anymore? maybe we need to store rootdir on the Suite though